deploy/crds/kieapp.crd.yaml

```diff
@@ -438,6 +438,13 @@ spec:
                           kieServerJmsAMQQueues:
                             type: string
                             description: AMQ broker broker comma separated queues, if empty the values from default queues will be used.
+                disablePrometheusExt:
+                  type: string
+                  description: An option indicating if prometheus server extension should be disabled
+                  enum:
+                    -
+                    - Enabled
+                    - Disabled
```

Comment:
Should be bool, possibly in a struct that's a pointer, see comment under kieapp_types